test/info.py

```diff
@@ -57,6 +57,7 @@
     ("k20dx_mimxrt1050_evk_hyper_if",               True,       0x8000,     "bin"       ),
     ("k20dx_mimxrt1050_evk_qspi_if",                True,       0x8000,     "bin"       ),
     ('k20dx_ep_agora_if',                           True,      	0x0000,     "bin"       ),
+    ('k20dx_ep_kairos_if'                           True,       0x0000,     "bin"       ),
```

Comment:
```suggestion
    ('k20dx_ep_kairos_if',                           True,       0x0000,     "bin"       ),
```

Reply:
@millerc-ep Can you make the suggested change?